AdminUi/apps/admin_ui/lib/home/identity_details/identity_messages/messages_data_table/recipients_dialog.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:admin_api_types/admin_api_types.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+import '/core/extensions.dart';
+
+class RecipientsDialog extends StatelessWidget {
+  final List<MessageRecipients> recipients;
+
+  const RecipientsDialog({required this.recipients, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(4),
+      child: FilledButton(
+        onPressed: () {
+          showDialog<Widget>(
+            context: context,
+            builder: (BuildContext context) {
+              return _RecipientsDialog(recipients: recipients);
+            },
+          );
+        },
+        child: const Text('...'),
+      ),
+    );
+  }
+}
```

Comment:
your "RecipientsDIALOG" is a button opening a dialog?! Do you see the issues about that? Please use modals and dialogs how we use dialogs and don't re-invent the wheel.

Reply:
an alternative would be to expose a ShowMoreRecipientsButton or ShowAllRecipientsButton (I don't know yet what this button is for) which then opens a dialog. If you do it like that you should rename the file.